ui/apps/platform/src/utils/pdfUtils.js

```diff
@@ -128,7 +128,7 @@ const createPDFTable = (tableData, entityType, query, pdfId, tableColumns) => {
                             String(flattenedObj[key]).replace(/\s+/g, ' ').trim()) ||
                         'N/A';
                 }
-                td.innerHTML = colValue;
+                td.innerHTML = colValue.replace(/<\/?[^>]+(>|$)/g, "");
```

Comment:
Instead of stripping string values, could we just set the content as text as-is?

```suggestion
                td.textContent = colValue;
```

Reply:
We could. However, I found that for some views, there are text formatting on the column value and sometimes icons being rendered. If we use the value as-is, it might look rather ugly for some PDF exports.

Reply:
Have you seen an example of this in practice with the applied fix? I did an analysis of this change and found that only a small subset of the columns hit this code path, and all of those columns appear to be strictly text-only. Any column containing an icon should be rendered as a React component and bypass this section entirely.

Reply:
I did see it during my testing, yes. I believe the react components gets transformed to _normal_ markup from [L121-L123](https://github.com/stackrox/stackrox/blob/f2daf17561088b1be37952b39ea081ca963e528f/ui/apps/platform/src/utils/pdfUtils.js#L121-L123)

Reply:
Ah, you are absolutely right! I was looking at this incorrectly. There are a few edge cases where we would lose some relatively insignificant formatting which I agree would be preferable to outputting the tags as text.

What do you think about moving the `td.innerHTML` inside the `if` branch above, and then use `td.textContent` in the `else` branch in [the section above](https://github.com/stackrox/stackrox/blob/f2daf17561088b1be37952b39ea081ca963e528f/ui/apps/platform/src/utils/pdfUtils.js#L120-L131)? That would leave the intended formatting intact in cases where the value was rendered with React, sanitize cases where we are expecting just a string value, as well as avoid edge cases such as removing a valid string containing the `<` character.